src/pybind/mgr/dashboard/frontend/src/app/shared/unit-test-helper.ts

```diff
@@ -0,0 +1,28 @@
+import { async, TestBed } from '@angular/core/testing';
+
+import { _DEV_ } from '../../unit-test-configuration';
+
+export class UnitTestHelper {
```

Comment:
You could export a static class or simply export the method, that way you don't have to initialize it every time on the spec files.